tools/internal_ci/linux/grpc_xds_php_test_in_docker.sh

```diff
@@ -60,10 +60,10 @@ export CC=/usr/bin/gcc
 
 GRPC_VERBOSITY=debug GRPC_TRACE=xds_client,xds_resolver,xds_routing_lb,cds_lb,eds_lb,priority_lb,weighted_target_lb,lrs_lb "$PYTHON" \
   tools/run_tests/run_xds_tests.py \
-  --test_case=all \
+  --test_case=ping_pong \
   --project_id=grpc-testing \
   --source_image=projects/grpc-testing/global/images/xds-test-server \
```

Comment:
Update this to `projects/grpc-testing/global/images/xds-test-server-2` (Append `-2`)
grpc/grpc-go#3764 (files)

Reply:
Done. Thanks for the pointer. Yea I also noticed the discrepancy.